viam-orb-slam3.go

```diff
@@ -499,7 +515,9 @@ func NewBuiltIn(ctx context.Context, deps registry.Dependencies, config config.S
 			modelName, svcConfig.ConfigParams["mode"])
 	}
 
-	slamConfig.SetupDirectories(svcConfig.DataDirectory, logger)
+	if err = slamConfig.SetupDirectories(svcConfig.DataDirectory, logger); err != nil {
+		return nil, errors.Errorf("unable to setup working directories: %v", err)
```

Comment:
Personal choice--while doing the tests for SetupDirectories I got familiar with the error messages generated by this function. In this case, this seemed like it would produce a nicer error message to do it this way. I have no strong reason to do it this way though and if you think it would be better to wrap, I am happy to swap to that.

Reply:
Both seem used, however after reading up a bit more on errors.Wrap(), I think you're right. I just swapped it.

Reply:
Sounds good, thanks for looking into it!